concent_api/concent_api/system_check.py

```diff
@@ -0,0 +1,29 @@
+from django.core.checks     import Error
+from django.core.checks     import register
+from django.core.checks     import Warning  # pylint: disable=redefined-builtin
+from django.conf            import settings
+
+
+@register()
+def check_settings_concent_features(app_configs, **kwargs):  # pylint: disable=unused-argument
+
+    if not isinstance(settings.CONCENT_FEATURES, list):
+        return [Error('The value of CONCENT_FEATURES setting is not a list', id = 'concent.E001')]
+
+    uknown_features = set(settings.CONCENT_FEATURES) - {"concent-api", "gatekeeper", "admin-panel"}
+    if len(uknown_features) > 0:
+        return [Error('Unknown features specified CONCENT_FEATURES setting',               hint = 'Did you make a typo in the name?', id = 'concent.E002')]
```

Comment:
Something went wrong with code formatting, there is a lot of extra spaces in this line and following ifs.

Reply:
Spaces where added by @cameel  for better readability.

Reply:
Splitting to multiple lines should do the job but as you guys want.